### ISSUE TITLE
Very long plugin name leads to github logo alignment issue

### ISSUE THREAD

**Issue by justinelarsen** [chanzuckerberg/napari-hub#272]
#### Description
When the plugin name is long, the github logo on the plugin detail page is vertically aligned to the middle and looks a little weird.

#### Steps/Code to Reproduce
Navigate to a plugin page where the plugin name is long. Under the "Source code" section, the name wraps onto two lines and the Github logo to the left is centered vertically. https://www.napari-hub.org/plugins/napari-segment-blobs-and-things-with-membranes

#### Expected Results
I think the github logo should be aligned vertically to the top instead of the middle, but would like @liaprins-czi 's input on how this should be.

#### Actual Results
![Screen Shot 2021-09-29 at 9 09 53 AM](https://user-images.githubusercontent.com/57509887/135309002-04d78f7b-ece0-4abd-aa80-bab953719ea1.png)

**Comment by liaprins-czi** [comment]
Yes, I agree with @justine's recommendation that the logo should be aligned to the top of the GH repo name. 

Also, the line height of the GH name looks too high; it should be 1.5 (let me knwo if I should make a separate issue for this!) Thanks!

**Comment by justinelarsen** [comment]
Thanks @liaprins-czi ! I think it's fine to include both updates in this issue since they are related and style updates. I created a task list at the top for both issues so it's clear there are two tasks.
